### DISCUSSION TITLE
5, 6번 단계 질문입니다.

### DISCUSSION THREAD

**Discussion by KyujinL** [changdoc/naver-vaccine-macro#115] [Q&A]
https://v-search.nid.naver.com/reservation/standby?orgCd=XXXX&sid=XXXXX 

위와 같은 형태에 알려 주신 방법으로 제가 알림신청한 병원의 id 값을 입력하고 엔터 누르니깐 

https://v-search.nid.naver.com/**reservation/auth?key**=XXX 이런 형태의 주소가 나옵니다. 
그런데 6번 내용을 보면 https://v-search.nid.naver.com/**reservation/info?key**=XXXX 이런 형태로 나와야 한다고 하셨는데 제가 갖게 된 값이랑 달라서요 

혹시 제가 어떤 과정을 누락했거나 한게 있을까요?

**Comment by changdoc** [comment]
네이버 로그인을 하고 진행하고 계신가요?

**Reply by KyujinL** [reply]
네네 네이버 로그인 하고 제가 지정한 병원 알림온게 있어서 거기서 지금 신청하기로 타고 들어갔는데 auth?key 이걸로 넘어가네요. 
그리고 네이버 인증이나 휴대폰 인증하라는 화면이 나옵니당

**Reply by KyujinL** [reply]
![image](https://user-images.githubusercontent.com/44232519/126964725-5cccc9f1-46ca-4ccd-8dff-7718659a7629.png)

사진처럼...
다른 분들은 보니깐 병원 명이랑 뭐 이런게 뜨던데 저는 인증하라는 화면 나와서 인증했는데 여전히 변하지 않고 저 상태에서 그냥 새로고침만 계속 하는 중이에요

**Reply by changdoc** [reply]
자동 예약 정지 하시고 인증 부터 정상적으로 진행완료 하시면 될 것 같네요.

**Reply by changdoc** [reply]
인증까지 완료 하시면 상단 화면이
![image](https://user-images.githubusercontent.com/1879494/126966062-041a97da-0365-4763-b1f4-eed4bdde1ab4.png)
이렇게 나올텐데요. 그때부터 매크로 동작시키시면 예약 가능 시점에 예약 시도 하게됩니다.

**Reply by KyujinL** [reply]
아... 다시 해보겠습니다 ㅜㅜ

**Reply by KyujinL** [reply]
왜인지는 모르겠으나 일단 엣지 브라우저로 하니깐 예약신청에 불이 들어와서 자동 예약 시작 눌러보긴 했습니다...내일 이렇게 다시 해봐야겠네요. 도움 말씀 주셔서 감사합니다.

**Reply by woobinn** [reply]
저도 크롬에선 저 창에서 안넘어가서 엣지로 돌리고 있습니다.
